src/distPlus/distribution/Continuous.re

```diff
@@ -208,48 +210,24 @@ let combineAlgebraicallyWithDiscrete =
     ) => {
   let t1s = t1 |> getShape;
   let t2s = t2.xyShape; // would like to use Discrete.getShape here, but current file structure doesn't allow for that
+
   let t1n = t1s |> XYShape.T.length;
   let t2n = t2s |> XYShape.T.length;
 
-  let fn = Operation.Algebraic.toFn(op);
-
-  let outXYShapes: array(array((float, float))) =
-    Belt.Array.makeUninitializedUnsafe(t2n);
-
-  for (j in 0 to t2n - 1) {
-    // for each one of the discrete points
-    // create a new distribution, as long as the original continuous one
+  if (t1n > 0 && t2n > 0) {
```

Comment:
I could imagine we could add some ``isEmpty`` method or similar. At this point, t1n is pretty terse to me.